DataConnectors/OrcaSecurityAlerts.json

```diff
@@ -0,0 +1,81 @@
+{
+    "id": "OrcaSecurityAlerts",
+    "title": "Orca Security Alerts",
+    "publisher": "Orca Security",
+    "descriptionMarkdown": "The Orca Security Alerts connector allows you to easily export Alerts logs to Azure Sentinel.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "OrcaAlerts_CL",
+            "baseQuery": "OrcaAlerts_CL"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Fetch all service vulnerabilities on running asset",
+            "query": "OrcaAlerts_CL | where alert_type_s == \"service_vulnerability\" | where asset_state_s == \"running\" | sort by TimeGenerated "
+        },
+        {
+            "description" : "Fetch all alerts with \"remote_code_execution\" label",
+            "query": "OrcaAlerts_CL | where split(alert_labels_s, \",\") contains(\"remote_code_execution\") | sort by TimeGenerated "
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "OrcaAlerts_CL",
+            "lastDataReceivedQuery": "OrcaAlerts_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "OrcaAlerts_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "write": true,
+                    "read": true,
+                    "delete": true
+                }
+            }
+        ]
+    },
+    "instructionSteps": [
+        {
+            "title": "",
+            "description":"Please see <a href='https://orcasecurity.zendesk.com/hc/en-us/articles/360043941992-Azure-Sentinel-configuration'>here</a> for integrating Orca Security Alerts logs with Azure Sentinel.",

```

Comment:
Please modify the text as follows...

Follow <a href='https://orcasecurity.zendesk.com/hc/en-us/articles/360043941992-Azure-Sentinel-configuration'> guidance </a> for integrating Orca Security Alerts logs with Azure Sentinel.

Reply:
Done.